README.md

```diff
@@ -0,0 +1,42 @@
+# Avidemux
+
+Avidemux is a simple cross-platform video editor for Linux, Windows and MacOsX.
+
+# Build Instructions
+
+Get the main repository:
+```
+git clone https://github.com/mean00/avidemux2.git
+```
+
+Get submodules:
+```
+git submodule update --init --recursive
+```
+
+## Debian / Ubuntu and derivatives
```

Comment:
I would prefer asking users to use existing convenience scripts. This is also a way to get bug reports for these scripts ;-)